src/main/js/controls/Gantt/Track.js

```diff
@@ -98,9 +99,14 @@ const addTrackLabelEventHandler = (canvasSVG, labelObj) =>
                     !utils.isFunction(labelObj.onClick)
                 );
             }
-            d3.select(this).on("click", () => {
-                labelObj.onClick(labelObj.display, d3.select(this));
-            });
+            if (
+                labelObj.display.length >
+                constants.DEFAULT_LABEL_CHARACTER_LIMIT
+            ) {
+                d3.select(this).on("click", () => {
```

Comment:
The cursor pointer is [shown](https://github.com/cerner/carbon-graphs/blob/master/src/main/less/controls/Gantt.less#L6) only when the consumer provides `onClick` property via the `aria-disabled` attribute. So if you allow the code to pass through Ln 99 then it would show the cursor pointer but when you click nothing would show up. 

Ideally, it should be:

Truncated: 
![image](https://user-images.githubusercontent.com/4908886/65714008-e9c49400-e05f-11e9-9858-bd7d6a3fffc6.png)

Non-truncated: 
![image](https://user-images.githubusercontent.com/4908886/65714125-20021380-e060-11e9-9a27-ebd0859b077e.png)

Reply:
Fixed here https://github.com/cerner/carbon-graphs/pull/89/commits/869cb84d5e749a87f211ddc7135beec7d007283c.